File: kodovi/PosiljkaWebApi_rad/Posiljka.Data/Helper/MyApiResult.cs
using System;

namespace Posiljka.Data.Helper
{
    public class MyApiResult<T> 
    {
        public String exceptionMessage = "";
        public bool isException = false;
        public int exceptionCode = 0;
        public T value;

        public static MyApiResult<T> Error(int exceptionCode, string exceptionMessage)
        {
            return new MyApiResult<T>
            {
                isException = true,
                exceptionCode = exceptionCode,
                exceptionMessage = exceptionMessage
            };
        }
        public static MyApiResult<T> OK(T value)
        {
            return new MyApiResult<T>
            {
                isException = false,
                value = value
            };
        }

       
    }

}

File: tmp/PosiljkaApp_rad/app/src/main/java/android/fit/ba/posiljka/helper/MyApiResult.java
package android.fit.ba.posiljka.helper;

/**
 * Created by Adil Joldic on 17.06.2017..
 */

public class MyApiResult<T> {
    public String errorMessage = "";
    public boolean isException = false;
    public int exceptionCode = 0;
    public T value;
}